build_tools/ci/build.py

```diff
@@ -305,7 +313,30 @@ def nvidia_gpu_build_with_compute_capability(
     configs=("warnings", "rbe_linux_cuda_nvcc"),
     compute_capability=75,
 )
+oneapi_tag_filter = (
+    "gpu",
+    "oneapi",
+    "requires-gpu-intel",
+    "-requires-gpu-amd",
+    "-requires-gpu-nvidia",
+    "-no_oss",
+    "-cuda-only",
+    "-rocm-only",
+    "-no-oneapi",
+)
 
+Build(
+    type_=BuildType.XLA_LINUX_X86_GPU_ONEAPI_GITHUB_ACTIONS,
+    repo="openxla/xla",
+    configs=("sycl", "sycl_hermetic"),
+    # This build of oneAPI backend runs on X86 host without an Intel GPU, so 
+    # build command is passed here instead of test.
+    subcommand="build",
```

Comment:
As mentioned above - if we filter test targets that require a GPU then we can set this to `test` and still run all the tests that don't require a GPU.